### ISSUE TITLE
Allow selection of pokéballs to use

### ISSUE THREAD

**Issue by JazzVT** [finndev/PokeBuddy#833]
Give us the option to choose wether to throw pokéballs, great balls or ultra balls. Now I set the item limit for pokéballs to 0, but if the bot doesn't transfer away the ones it just spinned before the next encounter, it will use the pokéballs and the chances of catching the pokémon are slim at best at higher levels.

**Comment by r0nalxd** [comment]
I think I read somewhere that the bot will be using different pokeballs when the probability is lower

**Comment by JazzVT** [comment]
Then maybe allow us to set when to use which pokeballs using a percentage?

**Comment by r0nalxd** [comment]
It could be implemented, just like the use of a razz berry can be adjusted in the item settings. Pls check #172 to see if this enhancement hasn't been suggested yet

**Comment by JazzVT** [comment]
It is indeed mentioned and has already been checked, this means it has been implemented? Then I suppose what was implemented is an automatich switch to different pokeballs at a certain percentage, not manual, which could be usefull. Or the option to only use great/ultra balls when farming pokéstops isn't an issue maybe.

**Comment by r0nalxd** [comment]
I assumed things checked in 172 are implemented to some degree

**Comment by LordNoxxious** [comment]
It needs to be a manual selection. The percentage based pokeball selection barely works.

**Comment by JazzVT** [comment]
Indeed. A selection for which pokeballs to use above certain CP's would be a good implementation.
